src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -76,38 +89,90 @@ protected void start(Map<String, String> props, ElasticsearchClient client) {
       log.warn("AK versions prior to 2.6 do not support the errant record reporter.");
     }
 
-    this.client = client != null ? client : new ElasticsearchClient(config, reporter);
+    this.client = client != null ? client
+            : new ElasticsearchClient(config, reporter, offsetTracker);
 
     log.info("Started ElasticsearchSinkTask. Connecting to ES server version: {}",
         getServerVersion());
   }
 
+  private static class PartitionPauser {
```

Comment:
It's probably better to move this to the beginning (before the member fields) or end (after the member methods) of this class.

Reply:
Moved to the end